### PR TITLE
gpio: Added split gpio addresses with their names.

### PR DESCRIPTION
**Pull request by HU90m**
This GPIO branch will be needed for testing https://github.com/lowRISC/sonata-system/pull/290

### CODE REVIEW

**Review by marnovandermaas — APPROVED** [review]
I think this is correct. Not sure if we want to hold off making this change until we have the rest of the pins in but approving to unblock.

**Comment by AlexJones0** [comment]
This is now out of date; as there should be `0x40` spacing between GPIO regions, not `0x20`. Probably better to do a more in-depth PR later as there is a lot of things to update in the board description file.

**Pull request closed without merging**
